docs/developer-docs/latest/setup-deployment-guides/installation/cli.md

```diff
@@ -38,84 +68,110 @@ Strapi currently support the following databases.
 | MariaDB    | 10.1            |
 | MongoDB    | 3.6             |
 
-## Step 2: Create a new project
+## Creating a Strapi project
```

Comment:
Is it worth it to add a link somewhere to redirect to this documentation?
https://strapi.io/documentation/developer-docs/latest/developer-resources/cli/CLI.html#strapi-new

Reply:
Same reply as [here](https://github.com/strapi/documentation/pull/326#discussion_r655394851)